src/http/routes/bucket/updateBucket.ts

```diff
@@ -44,9 +46,13 @@ export default async function routes(fastify: FastifyInstance) {
     async (request, response) => {
       const { bucketId } = request.params
 
-      const { public: isPublic } = request.body
+      const { public: isPublic, maxFileSizeKb, allowedMimeTypes } = request.body
```

Comment:
I think we need to assume all of these properties are optional now. If someone calls /update with just `public`, the existing values will get nuked right?

Reply:
these are optional, if not provided they will not change the underline value